### ISSUE TITLE
Problem: Some abci_query returns binary format which is hard to parse

### ISSUE THREAD

**Issue by yihuang** [crypto-com/chain#1464]
Currently, some paths in abci_query("account", "state") returns binary format.
I propose to change them into JSON, so a client can use them directly, rather than go through `client-rpc` or `client-cli`.

**Comment by yihuang** [comment]
An alternative is to use https://github.com/polkascan/py-scale-codec to decode, so chain-abci only need to concern about scale-codec encoding.

**Comment by tomtau** [comment]
one of reasons for choosing scale codec was that it has implementations in several languages:
https://github.com/polkadot-js/api/tree/master/packages/types/src/codec
https://github.com/ChainSafe/gossamer/tree/development/lib/scale
...

**Comment by yihuang** [comment]
> one of reasons for choosing scale codec was that it has implementations in several languages:
> https://github.com/polkadot-js/api/tree/master/packages/types/src/codec
> https://github.com/ChainSafe/gossamer/tree/development/lib/scale
> ...

Ok, I'll close this issue, we keep chain-abci sticking to SCALE codec, the only use JSON at client side.
